test/tests/pbs_smoketest.py

```diff
@@ -329,18 +329,29 @@ def test_limits(self):
     @skipOnCpuSet
     def test_finished_jobs(self):
         """
-        Test for finished jobs
+        Test for finished jobs and resource used for jobs.
         """
         a = {'resources_available.ncpus': '4'}
         self.server.manager(MGR_CMD_SET, NODE, a, self.mom.shortname)
         a = {'job_history_enable': 'True'}
         self.server.manager(MGR_CMD_SET, SERVER, a)
-        a = {'Resource_List.walltime': '10', ATTR_k: 'oe'}
-        j = Job(TEST_USER, attrs=a)
-        j.set_sleep_time(5)
+        a = {'Resource_List.select': '1:ncpus=2:mem=20mb',
+             'Resource_List.walltime': '15'}
+        j = Job(TEST_USER, a)
+        j.create_eatcpu_job(15)
```

Comment:
do we really need to increase the time from 5 to 15? the smoketest runs every time we execute a commit to github.

Reply:
based on the offline discussion - 
resources_used values are getting updated only after 10 seconds, so we will keep the time to be 15 seconds.